featurebyte/models/feature.py

```diff
@@ -551,6 +551,7 @@ class FeatureModel(BaseFeatureModel):
     aggregation_ids: List[str] = Field(allow_mutation=False, default_factory=list)
     aggregation_result_names: List[str] = Field(allow_mutation=False, default_factory=list)
     online_store_table_names: List[str] = Field(allow_mutation=False, default_factory=list)
+    agg_result_name_include_serving_names: bool = Field(default=False)  # backward compatibility
```

Comment:
nit: Maybe we should group those aggregation related attributes under a specific type, like `AggregationInfo`.

Reply:
Agree, that makes sense.